views/component/courseDetails/template.njk

```diff
@@ -54,7 +54,7 @@
         text: "Learning Provider"
       },
       {
-        text: "KPMG",
+        text: course.learningProvider.name | default("N/A", true),
```

Comment:
- We should only be showing LPs if the course contains face-to-face modules.
- Change `N/A` to `Unspecified`